### PR TITLE
Split the Extensions page in WP Admin and add respective menu items

### PR DESCRIPTION
**Pull request by madeincosmos**
Continuation of #30378 so that phpcs errors on the My Subscriptions page aren't a blocker for now.

### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
- Register submenu for My Subscriptions under WooCommerce
- Add a method in `WC_Admin_Addons` for Helper page output only
- remove top nav from the Extensions page (keeping it on the My Subscriptions page for now so phpcs is not a blocker)
- update e2e tests for Helper connection.

### How to test the changes in this Pull Request:

1. Check out this branch on local environment.
2. Go to WP Admin and observe there are two separate submenus for Extensions and My Subscriptions.
3. Open each of these pages and make sure they load correctly and have all styles in place.
4. Make sure the correct menu item is highlighted when you open each page.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Notes
- didn't remove top nav on the My Subscriptions page yet because editing this file triggered a lot of phpcs errors. I'll take care of it later.

### CODE REVIEW

File: includes/admin/class-wc-admin-menus.php
Comment (kloon) [inline includes/admin/class-wc-admin-menus.php:153]: Let's use this variable for My Subscriptions menu item, as it contains the subscription update count.

File: includes/admin/class-wc-admin-menus.php
Comment (kloon) [inline includes/admin/class-wc-admin-menus.php:154]: Title of this menu needs to be `Marketplace`

**Review by kloon — APPROVED** [review]
Changes look good, just left two things to change before we merge into the feature branch.

**Review by andfinally — COMMENTED** [review]
Looking great, thanks! I think we need to make some additional changes to the My Subscriptions page, http://localhost:8100/wp-admin/admin.php?page=wc-addons&section=helper. Apologies for not spelling these out in the issue.

* The page title at the top of the page should be "My Subscriptions" instead of "Extensions".
* We should remove the tab bar on this page, as the "Browse Extensions" page is now the separate Marketplace page.
* We should replace the h2 "Subscriptions" with "WooCommerce.com Subscriptions".

Also, on the Marketplace page, the page title at the top needs to be Marketplace, as well as the menu item Gerhard mentioned.

image

image

File: includes/admin/class-wc-admin-menus.php
Comment (madeincosmos) [reply includes/admin/class-wc-admin-menus.php]: Thanks, updated it now 🙌

**Comment by kloon** [comment]
The titles need to be adjusted in WC Admin

https://github.com/woocommerce/woocommerce-admin/blob/38fbb4aedd0f21cf21ddaa6af93e48564fd35324/includes/connect-existing-pages.php#L24

File: includes/admin/class-wc-admin-menus.php
Comment (madeincosmos) [reply includes/admin/class-wc-admin-menus.php]: Renamed to Marketplace.

**Comment by madeincosmos** [comment]
All feedback on this PR is addressed now except for Marketplace and My Subscriptions page headers than need to be set in WooCommerce Admin instead. 

I'm going to merge this one once the PHPCS issues are addressed in #30405.

File: includes/admin/views/html-admin-page-addons.php
Comment (kloon) [inline includes/admin/views/html-admin-page-addons.php:16]: Let's change this to `WooCommerce Marketplace` to match the title, this is only for screen readers but should match the main page title.

File: includes/admin/class-wc-admin-menus.php
Comment (kloon) [inline includes/admin/class-wc-admin-menus.php:154]:
```suggestion
		add_submenu_page( 'woocommerce', __( 'WooCommerce Marketplace', 'woocommerce' ), __( 'Marketplace', 'woocommerce' ), 'manage_woocommerce', 'wc-addons', array( $this, 'addons_page' ) );
```
Let's change the browser title to match the actual title on the page.

**Review by kloon — APPROVED** [review]
Latest changes works great, thanks @madeincosmos.

I am approving assume you can just address the two small title changes so we have consistent titles.

File: includes/admin/views/html-admin-page-addons.php
Comment (madeincosmos) [reply includes/admin/views/html-admin-page-addons.php:16]: Thanks, I updated the title now.

File: includes/admin/class-wc-admin-menus.php
Comment (madeincosmos) [reply includes/admin/class-wc-admin-menus.php:154]: Updated ✅

**Review by andfinally — APPROVED** [review]
Thanks for this excellent, neat solution! 👌

**Comment by andfinally** [comment]
For some reason I'm still getting "WooCommerce extensions" as the browser tab title on both pages, I think this may be related to the issue you [referred](p1628245850167600-slack-C01LE21AWCD) to.

**Pull request merged**
